recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -3,7 +3,9 @@ FILESEXTRAPATHS_prepend := "${THISDIR}:"
 SRC_URI += "file://meson64-kmeta;type=kmeta;destsuffix=meson64-kmeta"
 
 COMPATIBLE_MACHINE_khadas-vim3 = "khadas-vim3"
+COMPATIBLE_MACHINE_khadas-vim3-sdboot = "khadas-vim3-sdboot"
```

Comment:
At some point we should allow all board sur use this recipe, so something like:
COMPATIBLE_MACHINE_meson-gx = "$MACHINE"

Reply:
Agree, but I don't have access to any of the other machines so can't test it.

Reply:
No problem, since it's not the default kernel it's not an issue